### PR TITLE
Zaraz Monitoring API

### PR DESCRIPTION
**Pull request by bjesus**

### CODE REVIEW

File: src/content/docs/zaraz/monitoring/monitoring-api.mdx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+## Example GraphQL Queries
+
+You can construct any query you'd like using the above datasets, but here are some example queries you can use.
+
+<Tabs syncKey="GQLExamples"><TabItem label="Events">
+
+Query for the count of Zaraz events, grouped by time.
+
+```graphql
+query AllEvents(
```
Comment (vicb) [inline src/content/docs/zaraz/monitoring/monitoring-api.mdx]:
```suggestion
query ZarazEvents(
```
for consistency

File: src/content/docs/zaraz/monitoring/monitoring-api.mdx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+ }
+}
+```
+
+</TabItem><TabItem label="Erroneous responses">
+
+Query for the count of 400 server-side responses, grouped by time and URL.
+
+```graphql
+query SucessfulGA4Requests(
```
Comment (vicb) [inline src/content/docs/zaraz/monitoring/monitoring-api.mdx]: Not sure about the name here?

File: src/content/docs/zaraz/monitoring/monitoring-api.mdx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+) {
+ viewer {
+  zones(filter: { zoneTag: $zoneTag }) {
+   data: zarazFetchAdaptiveGroups(
+    limit: $limit
+    filter: {
+     datetimeHour_geq: $start
+     datetimeHour_leq: $end
+     url_neq: ""
+     OR: [{ AND: [{ status: 400 }] }]
```
Comment (vicb) [inline src/content/docs/zaraz/monitoring/monitoring-api.mdx]: I have no clue about graphql, but do we really need OR and AND ?

File: src/content/docs/zaraz/monitoring/monitoring-api.mdx [hunk 4]
```diff
@@ -0,0 +1,202 @@
+---
+pcx_content_type: how-to
+title: Monitoring API
+sidebar:
+  order: 2
+---
+
+import { TabItem, Tabs } from "~/components";
+
+The **Zaraz Monitoring API** allows users to retrieve detailed data on Zaraz events through the **GraphQL Analytics API**. Using this API, you can monitor events, pageviews, triggers, actions, and server-side request statuses, including any errors and successes. The data available through the API mirrors what is shown on the Zaraz Monitoring page in the dashboard, but with the API, you can query it programmatically to create alerts and notifications for unexpected deviations
```
Comment (vicb) [inline src/content/docs/zaraz/monitoring/monitoring-api.mdx]:
```suggestion
The **Zaraz Monitoring API** allows users to retrieve detailed data on Zaraz events through the **GraphQL Analytics API**. Using this API, you can monitor events, pageviews, triggers, actions, and server-side request statuses, including any errors and successes. The data available through the API mirrors what is shown on the Zaraz Monitoring page in the dashboard, but with the API, you can query it programmatically to create alerts and notifications for unexpected deviations.
```

**Review by simonabadoiu — APPROVED** [review]

File: src/content/docs/zaraz/monitoring/monitoring-api.mdx (hunk 3 above)
Comment (bjesus) [reply src/content/docs/zaraz/monitoring/monitoring-api.mdx]: should be fixed now! thanks for pointing it out

File: src/content/docs/zaraz/monitoring/monitoring-api.mdx (hunk 2 above)
Comment (bjesus) [reply src/content/docs/zaraz/monitoring/monitoring-api.mdx]: fixed

**Pull request merged**